src/validation/index.ts

```diff
@@ -1,12 +1,6 @@
-export interface ValidationMessage {
-  key: string
-}
-
-export interface Validation {
-  errors: Array<ValidationMessage>
-  fields: {
-    [name: string]: Validation
-  }
-  valid: boolean
-  warnings: Array<ValidationMessage>
-}
+export type { FieldValidator } from './fieldValidator'
+export { FieldValidators } from './fieldValidators'
+export type { Validation } from './validation'
+export { ValidationResult, ValidationSeverity } from './validation'
```

Comment:
ValidationResult must be exported with type

Reply:
yes, sorry